<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="UM Bioinformatics Core" />

<meta name="date" content="2023-08-21" />

<title>Bulk ATAC-Seq</title>

<script src="site_libs/header-attrs-2.24/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link rel="shortcut icon" href="favicon-16x16.png" />
<!--
Favicon from
https://twemoji.twitter.com/
https://favicon.io/emoji-favicons/dna/
-->

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Analysis workflows and methods</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    RNA-Seq
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="bulk-rna-seq.html">Bulk RNA-Seq</a>
    </li>
    <li>
      <a href="single-cell-rna-seq.html">Single Cell RNA-Seq</a>
    </li>
    <li>
      <a href="single-nuclei-rna-seq.html">Single Nuclei RNA-Seq</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Epigenomics
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="bulk-atac-seq.html">ATAC-Seq</a>
    </li>
    <li>
      <a href="bulk-chip-seq.html">ChIP-seq</a>
    </li>
    <li>
      <a href="bulk-cut-and.html">Cut and Run</a>
    </li>
    <li>
      <a href="dna-methylation-epic.html">Methylation (EPIC)</a>
    </li>
    <li>
      <a href="dna-methylation-seq.html">Methylation (Sequencing)</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Bulk ATAC-Seq</h1>
<h4 class="author">UM Bioinformatics Core</h4>
<h4 class="date">2023-08-21</h4>

</div>


<style type="text/css">
body, td {
   font-size: 18px;
}
code.r{
  font-size: 12px;
}
pre {
  font-size: 12px
}
</style>
<p><img src="images/Orochi_atac.png" alt="ATAC-Seq workflow" style="width: 1000px;"/></p>
<div id="methods" class="section level2">
<h2>Methods</h2>
<div id="qc" class="section level3">
<h3>QC</h3>
<p>We use FastQC <span class="citation">(<span>FastQC</span>)</span>
(v0.11.8) to assess the overall quality of each sequenced sample.</p>
<p>We use TrimGalore <span
class="citation">(<span>TrimGalore</span>)</span> (v0.4.5) and cutadapt
<span class="citation">(Martin)</span> (v1.15) with the following
parameters:
<code>--nextera -e 0.1 --stringency 6 --length 20 --nextseq 20</code>.</p>
<p>We align trimmed reads to the reference genome with Bowtie2 <span
class="citation">(Langmead and Salzberg, 2012)</span> (v2.3.4.1) with
the following parameters:
<code>-X 2000 --no-mixed --no-discordant</code>, and defaults multi-seed
length of 20bp with 0 mismatches.</p>
<p>Duplicate reads are marked with Picard <span
class="citation">(Picard)</span> (v2.20.2).</p>
<p>Alignments to autosomes and sex chromosomes are kept
(i.e. mitochondrial reads are removed), duplicates marked by Picard are
removed, and alignments below a MAPQ threshold are removed. These
filtering steps are performed with samtools <span class="citation">(Li
<em>et al.</em>, 2009)</span> (v1.2) and the parameters:
<code>-q 10 -F 1024</code>.</p>
<p>Reads completely overlapping blacklisted regions <span
class="citation">(<span>ENCODE</span> <span>Blacklist</span>
<span>Regions</span>)</span> are removed with bedtools <span
class="citation">(Quinlan and Hall, 2010)</span> (v2.28.0).</p>
<p>Sample-wise peaks are called with macs2 <span class="citation">(Zhang
<em>et al.</em>, 2008)</span> (v2.1.2) with flags:
<code>-f BAM --nomodel --shift -100 --extsize 200</code>.</p>
<p>Peaks over all samples are merged with bedops <span
class="citation">(Neph <em>et al.</em>, 2012)</span> (v2.4.36) for the
purpose of principal component analysis and unsupervised clustering to
assess the similarity of samples.</p>
<p>Finally, MultiQC <span class="citation">(Ewels <em>et al.</em>,
2016)</span> (v1.7) generates a report combining FastQC, trimming,
alignment, and duplicate calling over all the samples. For ATAC specific
QC metrics we use ataqv <span class="citation">(Ataqv)</span>
(v1.0.0).</p>
</div>
<div id="differential-testing" class="section level3">
<h3>Differential Testing</h3>
<p>We use the edgeR R Bioconductor package <span
class="citation">(McCarthy <em>et al.</em>, 2012)</span> to identify
regions of differentially open chromatin (DOC). For each sample, the
number of reads in the merged peaks is counted for each sample, and a
library size normalization factor is determined. With no replicates, we
manually tune the BCV (biological coefficient of variance) parameter. We
then fit each model using the glmFit function, and test each contrast
with a likelihood ratio test. With replicates, the common, trended, and
tagwise negative binomial dispersions are calculated. We then fit each
model using the glmQLFit function, and test each contrast with an
empirical Bayes quasi-likelihood F-test. The DOC are then annotated to
genic and CpG island annotations using the annotatr R Bioconductor
package <span class="citation">(Cavalcante and Sartor, 2017)</span>.</p>
</div>
<div id="footprinting" class="section level3">
<h3>Footprinting</h3>
<p>We use the HINT tool in the Regulatory Genomics Toolbox (rgt-hint) to
do footprinting and motif binding analysis <span class="citation">(Li
<em>et al.</em>, 2019)</span>. Briefly, for each condition, the reads
among the replicates are pooled and tag counts are determined within the
merged peaks. Next motifs from the JASPAR database <span
class="citation">(Khan <em>et al.</em>, 2017)</span> are queried against
the footprints found within each condition. Finally, a differential
score representing the transcription factor binding activity and the
openness of the surrounding chromatin is calculated and visualized
between pairs of conditions.</p>
</div>
</div>
<div id="references" class="section level2 unnumbered">
<h2 class="unnumbered">References</h2>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-noauthor_ataqv_nodate" class="csl-entry">
<a href="https://github.com/ParkerLab/ataqv">Ataqv</a>
<em>https://github.com/ParkerLab/ataqv</em>.
</div>
<div id="ref-cavalcante_annotatr_2017" class="csl-entry">
Cavalcante,R.G. and Sartor,M.A. (2017) <a
href="https://doi.org/10.1093/bioinformatics/btx183">Annotatr: Genomic
regions in context</a>. <em>Bioinformatics</em>, <strong>33</strong>,
2381–2383.
</div>
<div id="ref-noauthor_encode_nodate" class="csl-entry">
<a
href="https://sites.google.com/site/anshulkundaje/projects/blacklists"><span>ENCODE</span>
<span>Blacklist</span> <span>Regions</span></a>
<em>https://sites.google.com/site/anshulkundaje/projects/blacklists</em>.
</div>
<div id="ref-ewels_multiqc_2016" class="csl-entry">
Ewels,P. <em>et al.</em> (2016) <a
href="https://doi.org/10.1093/bioinformatics/btw354"><span>MultiQC</span>:
Summarize analysis results for multiple tools and samples in a single
report</a>. <em>Bioinformatics</em>, <strong>32</strong>, 3047–3048.
</div>
<div id="ref-noauthor_fastqc_nodate" class="csl-entry">
<a href="https://github.com/s-andrews/FastQC"><span>FastQC</span></a>
<em>https://github.com/s-andrews/FastQC</em>.
</div>
<div id="ref-jaspar" class="csl-entry">
Khan,A. <em>et al.</em> (2017) <a
href="https://doi.org/10.1093/nar/gkx1126"><span class="nocase">JASPAR
2018: update of the open-access database of transcription factor binding
profiles and its web framework</span></a>. <em>Nucleic Acids
Research</em>, <strong>46</strong>, D260–D266.
</div>
<div id="ref-langmead_fast_2012" class="csl-entry">
Langmead,B. and Salzberg,S.L. (2012) <a
href="https://doi.org/10.1038/nmeth.1923">Fast gapped-read alignment
with <span>Bowtie</span> 2</a>. <em>Nature Methods</em>,
<strong>9</strong>, 357–359.
</div>
<div id="ref-li_sequence_2009" class="csl-entry">
Li,H. <em>et al.</em> (2009) <a
href="https://doi.org/10.1093/bioinformatics/btp352">The
<span>Sequence</span> <span>Alignment</span>/<span>Map</span> format and
<span>SAMtools</span></a>. <em>Bioinformatics</em>, <strong>25</strong>,
2078–2079.
</div>
<div id="ref-li_hint_2019" class="csl-entry">
Li,Z. <em>et al.</em> (2019) <a
href="https://doi.org/10.1186/s13059-019-1642-2">Identification of
transcription factor binding sites using ATAC-seq</a>. <em>Genome
Biology</em>, <strong>20</strong>.
</div>
<div id="ref-martin_cutadapt_nodate" class="csl-entry">
Martin,M. <a href="https://doi.org/10.14806/ej.17.1.200">Cutadapt
removes adapter sequences from high-throughput sequencing reads</a>.
<em>EMBnet.journal</em>, <strong>17</strong>, 3.
</div>
<div id="ref-mccarthy_differential_2012" class="csl-entry">
McCarthy,D.J. <em>et al.</em> (2012) <a
href="https://doi.org/10.1093/nar/gks042">Differential expression
analysis of multifactor <span>RNA</span>-<span>Seq</span> experiments
with respect to biological variation</a>. <em>Nucleic Acids
Research</em>, <strong>40</strong>, 4288–4297.
</div>
<div id="ref-neph_bedops_2012" class="csl-entry">
Neph,S. <em>et al.</em> (2012) <a
href="https://doi.org/10.1093/bioinformatics/bts277"><span>BEDOPS</span>:
High-performance genomic feature operations</a>.
<em>Bioinformatics</em>, <strong>28</strong>, 1919–1920.
</div>
<div id="ref-noauthor_picard_nodate" class="csl-entry">
<a href="https://github.com/broadinstitute/picard">Picard</a>
<em>https://github.com/broadinstitute/picard</em>.
</div>
<div id="ref-quinlan_bedtools_2010" class="csl-entry">
Quinlan,A.R. and Hall,I.M. (2010) <a
href="https://doi.org/10.1093/bioinformatics/btq033"><span>BEDTools</span>:
A flexible suite of utilities for comparing genomic features</a>.
<em>Bioinformatics</em>, <strong>26</strong>, 841–842.
</div>
<div id="ref-noauthor_trimgalore_nodate" class="csl-entry">
<a
href="https://github.com/FelixKrueger/TrimGalore"><span>TrimGalore</span></a>
<em>https://github.com/FelixKrueger/TrimGalore</em>.
</div>
<div id="ref-zhang_model-based_2008" class="csl-entry">
Zhang,Y. <em>et al.</em> (2008) <a
href="https://doi.org/10.1186/gb-2008-9-9-r137">Model-based
<span>Analysis</span> of <span>ChIP</span>-<span>Seq</span>
(<span>MACS</span>)</a>. <em>Genome Biology</em>, <strong>9</strong>,
R137.
</div>
</div>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3,h4",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
