<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="UM Bioinformatics Core" />

<meta name="date" content="2023-08-29" />

<title>Single Cell/Nuclei RNA-Seq</title>

<script src="site_libs/header-attrs-2.21/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link rel="shortcut icon" href="favicon-16x16.png" />
<!--
Favicon from
https://twemoji.twitter.com/
https://favicon.io/emoji-favicons/dna/
-->

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Analysis workflows and methods</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Gene Expression
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="bulk-rna-seq.html">Bulk RNA-Seq</a>
    </li>
    <li>
      <a href="scsn-rna-seq.html">Single Cell/Nuclei RNA-Seq</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Epigenomics
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="bulk-atac-seq.html">ATAC-Seq</a>
    </li>
    <li>
      <a href="bulk-chip-seq.html">ChIP-seq</a>
    </li>
    <li>
      <a href="bulk-cut-and.html">Cut and Run</a>
    </li>
    <li>
      <a href="dna-methylation-epic.html">Methylation (EPIC)</a>
    </li>
    <li>
      <a href="dna-methylation-seq.html">Methylation (Sequencing)</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Spatial
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="spatial-GeoMX.html">Nanostring GeoMX</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Single Cell/Nuclei RNA-Seq</h1>
<h4 class="author">UM Bioinformatics Core</h4>
<h4 class="date">2023-08-29</h4>

</div>


<style type="text/css">
body, td {
   font-size: 18px;
}
code.r{
  font-size: 12px;
}
pre {
  font-size: 12px
}
</style>
<div class="float">
<img src="images/scsn-rna-seq-workflow.png"
alt="Single cell/nuclei RNA-Seq Workflow" />
<div class="figcaption">Single cell/nuclei RNA-Seq Workflow</div>
</div>
<div id="methods" class="section level2">
<h2>Methods</h2>
<p>Raw sequence data is transformed to gene-barcode count matrices using
CellRanger <span class="citation">(Zheng <em>et al.</em>, 2017)</span>.
Further data analysis is primarily performed using the Seurat package
<span class="citation">(Stuart <em>et al.</em>, 2019)</span>.
Gene-barcode matrices and metadata for each sample are loaded and
further filtering and clustering analyses were performed as described in
the Seurat tutorials <span class="citation">(<span class="nocase">Seurat
tutorials</span>)</span>. Aberrant cells are filtered (low complexity,
duplets, or apoptotic cells) and based on detected debris/contamination
DecontX <span class="citation">(Yang <em>et al.</em>, 2020)</span> may
be run. Counts are normalized using the default normalization approach
and variable features were identified. Where appropriate, anchor points
were then generated across related datasets and used for SCTransform
data integration. Principal component analysis (PCA) is then performed
to identify significant PCA components used to find nearest neighbors
followed by graph-based, semi- unsupervised clustering into distinct
populations. Projections are generated using uniform manifold
approximation <span class="citation">(Becht <em>et al.</em>,
2018)</span> and marker genes are identified through differential gene
expression pairwise comparisons (Wilcoxon rank-sum test for single-cell
gene expression; FindAllMarkers function). Cell-type predictions were
also generated with scCATCH <span class="citation">(Shao <em>et
al.</em>, 2020)</span>. Suitability and approach for trajectory analysis
is determined based on experimental design and disease model <span
class="citation">(Saelens <em>et al.</em>, 2019; Lange <em>et al.</em>,
2022; Cao <em>et al.</em>, 2019; Wolf <em>et al.</em>, 2019; La Manno
<em>et al.</em>, 2018; Bergen <em>et al.</em>, 2020)</span>.</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2 class="unnumbered">References</h2>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-Becht_UMAP_2018" class="csl-entry">
Becht,E. <em>et al.</em> (2018) <a
href="https://doi.org/10.1038/nbt.4314">Dimensionality reduction for
visualizing single-cell data using UMAP</a>. <em>Nature
Biotechnology</em>.
</div>
<div id="ref-Bergen_RNAVelocity_2020" class="csl-entry">
Bergen,V. <em>et al.</em> (2020) <a
href="https://doi.org/10.1038/s41587-020-0591-3">Generalizing RNA
velocity to transient cell states through dynamical modeling</a>.
<em>Nature Biotechnology</em>, <strong>38</strong>, 1408–1414.
</div>
<div id="ref-Cao_Monocle3_2019" class="csl-entry">
Cao,J. <em>et al.</em> (2019) <a
href="https://doi.org/10.1038/s41586-019-0969-x">The single-cell
transcriptional landscape of mammalian organogenesis</a>.
<em>Nature</em>, <strong>566</strong>, 496–502.
</div>
<div id="ref-La_Manno_RNAVelocity_2018" class="csl-entry">
La Manno,G. <em>et al.</em> (2018) <a
href="https://doi.org/10.1038/s41586-018-0414-6">RNA velocity of single
cells</a>. <em>Nature</em>, <strong>560</strong>, 494–498.
</div>
<div id="ref-Lange_CellRank_2022" class="csl-entry">
Lange,M. <em>et al.</em> (2022) <a
href="https://doi.org/10.1038/s41592-021-01346-6">CellRank for directed
single-cell fate mapping</a>. <em>Nature Methods</em>,
<strong>19</strong>, 159–170.
</div>
<div id="ref-Saelens_2019" class="csl-entry">
Saelens,W. <em>et al.</em> (2019) <a
href="https://doi.org/10.1038/s41587-019-0071-9">A comparison of
single-cell trajectory inference methods</a>. <em>Nature
Biotechnology</em>, <strong>37</strong>, 547–554.
</div>
<div id="ref-Seurat_tutorials" class="csl-entry">
<a href="http://satijalab.org/seurat/"><span class="nocase">Seurat
tutorials</span></a>.
</div>
<div id="ref-Shao_scCATCH_2020" class="csl-entry">
Shao,X. <em>et al.</em> (2020) <a
href="https://doi.org/10.1016/j.isci.2020.100882">scCATCH: Automatic
annotation on cell types of clusters from single-cell RNA sequencing
data</a>. <em>iScience</em>, <strong>23</strong>, 100882.
</div>
<div id="ref-Stuart_Seurat_2019" class="csl-entry">
Stuart,T. <em>et al.</em> (2019) <a
href="https://doi.org/10.1016/j.cell.2019.05.031">Comprehensive
integration of single-cell data</a>. <em>Cell</em>,
<strong>177</strong>, 1888–1902.e21.
</div>
<div id="ref-Wolf_PAGA_2019" class="csl-entry">
Wolf,F.A. <em>et al.</em> (2019) <a
href="https://doi.org/10.1186/s13059-019-1663-x">PAGA: Graph abstraction
reconciles clustering with trajectory inference through a topology
preserving map of single cells</a>. <em>Genome Biology</em>,
<strong>20</strong>, 59.
</div>
<div id="ref-Yang_DecontX_2020" class="csl-entry">
Yang,S. <em>et al.</em> (2020) <a
href="https://doi.org/10.1186/s13059-020-1950-6">Decontamination of
ambient RNA in single-cell RNA-seq with DecontX</a>. <em>Genome
Biology</em>, <strong>21</strong>, 57.
</div>
<div id="ref-Zheng_CellRanger_2017" class="csl-entry">
Zheng,G.X.Y. <em>et al.</em> (2017) <a
href="https://doi.org/10.1038/ncomms14049">Massively parallel digital
transcriptional profiling of single cells</a>. <em>Nature
Communications</em>, <strong>8</strong>, 14049.
</div>
</div>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3,h4",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
